.github/workflows/ci.yml

```diff
@@ -36,17 +36,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: p0deje/setup-bazel@0.1.0
+      - run: bazel test ...
```

Comment:
This would run as a matrix for all Ruby versions - we don't want that.

Let's get it back to a separate job, see my suggestion below.